spring.core/src/main/java/com/epam/springcore/task/storage/StorageInitializer.java

```diff
@@ -0,0 +1,65 @@
+package com.epam.springcore.task.storage;
+
+
+import com.epam.springcore.task.model.JsonDataContainer;
+import com.epam.springcore.task.model.Trainee;
+import com.epam.springcore.task.model.Trainer;
+import com.epam.springcore.task.model.Training;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import jakarta.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.text.SimpleDateFormat;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Component
+public class StorageInitializer {
```

Comment:
Use application startup event listener
https://www.baeldung.com/running-setup-logic-on-startup-in-spring